operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1602,3 +1602,31 @@ gateway:
       - secretName: gem-gateway-tls
         hosts:
           - gateway.gem.example.com
+
+# Rules for the Prometheus Operator
+prometheusRule:
+  # -- If enabled, a PrometheusRule resource for Prometheus Operator is created
+  enabled: false
+  # -- PrometheusRule annotations
+  annotations: {}
+  # -- Additional PrometheusRule labels
+  labels: {}
```

Comment:
let's add namespace as option , in case operator is not scanning all namespaces, see my other comment